native/Avalonia.Native/src/OSX/AvnView.mm

```diff
@@ -290,6 +290,37 @@ - (void)mouseEvent:(NSEvent *)event withType:(AvnRawMouseEventType) type
         delta.Y = [event deltaY];
     }
 
+    float pressure = 0.0f;
+    float xTilt = 0.0f;
+    float yTilt = 0.0f;
+    AvnPointerDeviceType pointerType = AvnPointerDeviceType::Mouse;
+
+    switch(event.subtype)
+    {
+        case NSEventSubtypeTabletPoint:
+            switch(event.type)
+            {
+                case NSEventTypeLeftMouseDown:
+                case NSEventTypeLeftMouseDragged:
+                case NSEventTypeRightMouseDown:
+                case NSEventTypeRightMouseDragged:
+                case NSEventTypeOtherMouseDown:
+                case NSEventTypeOtherMouseDragged:
+                    pressure = event.pressure;
+                    break;
+                default:
+                    pressure = 0.0f;
+                    break;
+            }
+            xTilt =  event.tilt.x * 90;
+            yTilt = -event.tilt.y * 90;
```

Comment:
For better understanding, please set the `pointerType` to `Pen` here and add a `break` instead of relying on the implicit `case` fallback.